Testing/SystemTests/tests/framework/ISIS/SANS/WORKFLOWS/SANSBatchReductionTest.py

```diff
@@ -39,6 +41,15 @@ def _compare_workspace(self, workspace, reference_file_name):
         load_alg.execute()
         reference_workspace = load_alg.getProperty("OutputWorkspace").value
 
+        # Save the workspace out
+        f_name = path.join(config.getString('defaultsave.directory'),
```

Comment:
Just so I understand. Is this here because the validation is done through some other means so a mismatch file is not saved?

Reply:
Yeah some of these tests weren't saving out files if they failed, which made diagnostics frustrating.